coder/main.go

```diff
@@ -170,9 +187,36 @@ func Run(s *discordgo.Session, m *discordgo.MessageCreate) error {
 	//cli, err := client.NewEnvClient()
 	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
 	defer cancel()
+	
+	//Has attachments files
+	if(len(m.Attachments) > 0){
+		lang:="undefined";
+
+		
+		isPy,_ :=regexp.MatchString(`\.py?`, m.Attachments[0].URL)
+		isJs, _ :=regexp.MatchString(`\.js?`,m.Attachments[0].URL)
+		//ADD more laguages here	
```

Comment:
There is already a function for dealing with languages, `parseLang` and `langToImage`, which should be reused for this functionality. That way there won't be multiple places to maintain for the available alnguages.